backend/src/v5/middleware/dataConverter/inputs/teamspaces/projects/models/commons/tickets.comments.js

```diff
@@ -16,29 +16,39 @@
  */
 
 const { createResponseCode, templates } = require('../../../../../../../utils/responseCodes');
+const { types, utils: { stripWhen } } = require('../../../../../../../utils/helper/yup');
 const { UUIDToString } = require('../../../../../../../utils/helper/uuids');
 const Yup = require('yup');
 const { getCommentById } = require('../../../../../../../models/tickets.comments');
 const { getTicketById } = require('../../../../../../../models/tickets');
 const { getUserFromSession } = require('../../../../../../../utils/sessions');
+const { isEqual } = require('../../../../../../../utils/helper/objects');
 const { isUUIDString } = require('../../../../../../../utils/helper/typeCheck');
 const { respond } = require('../../../../../../../utils/responder');
-const { types } = require('../../../../../../../utils/helper/yup');
 const { validateMany } = require('../../../../../../common');
 
 const CommentsMiddleware = {};
 
 const validateComment = (isNewComment) => async (req, res, next) => {
 	try {
 		const schema = Yup.object().shape({
-			comment: types.strings.longDescription,
+			comment: isNewComment
+				? types.strings.longDescription
+				: stripWhen(types.strings.longDescription, (t) => isEqual(t, req.commentData.comment)),
```

Comment:
I don't think that's the right approach.. 😆 

Just let the comment validate and check if the comment string/image array are the same. Also this would in theory remove the comment string if I changed the image without changing the comment string?

The `stripWhen` just complicates the schema unnecessarily, and you're also not catching the situation when the image array is the same (i.e. if i pass in the same references)